ComplianceSuite/Makefile

```diff
@@ -11,13 +11,20 @@ fmt:
 lint:
 	swift format lint --parallel --recursive .
 
+.PHONY: test
+test:
+	mkdir -p .build/test-results
+	swift test \
+	--filter ComplianceTests.ParseDurationTests \
+	--xunit-output .build/test-results/junit.xml
+
 .PHONY: test-compliance
 test-compliance:
-	mkdir -p .build/test-results
+	mkdir -p .build/test-compliance-results
```

Comment:
❓Are these changes compatible with our new CI runners?

Reply:
Yes, I think so, but if we are simply combining compliance suite and regular tests in ComplianceSuite package into one test target, this is unnecessary.